Nodejs/Product/Nodejs/Debugger/DebugEngine/Remote/NodeRemoteEnumDebugProcesses.cs

```diff
@@ -103,18 +103,24 @@ public NodeRemoteEnumDebugProcesses(NodeRemoteEnumDebugProcesses processes)
                         break;
                     }
                 } catch (OperationCanceledException) {
-                    LiveLogger.WriteLine("NodeRemoteEnumDebugProcesses ping timed out.");
+                    LiveLogger.WriteLine("OperationCanceledException connecting to remote debugger");
```

Comment:
We included the "ping timed out" before to make it clearer what the core issue actually was. Don't mind including the exception as well, but we should also include the ping timeout bit.